task/01-strings-tasks.js

```diff
@@ -238,7 +295,11 @@ function encodeToRot13(str) {
  *   isString(new String('test')) => true
  */
 function isString(value) {
-    throw new Error('Not implemented');
+    for (const valueKey in value) {
+        return typeof value[valueKey] === 'string';
+    }
```

Comment:
What is it?
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof